graphql_jwt/refresh_token/mixins.py

```diff
@@ -26,12 +28,15 @@ def refresh(cls, root, info, refresh_token, **kwargs):
 
         payload = jwt_settings.JWT_PAYLOAD_HANDLER(refresh_token.user, context)
         token = jwt_settings.JWT_ENCODE_HANDLER(payload, context)
-
         refresh_token.rotate(context)
-        refreshed_token = refresh_token_lazy(refresh_token.user)
 
         if getattr(context, 'jwt_cookie', False):
```

Comment:
@mongkok How would this ever evaluate to true?  I don't see `jwt_cookie` ever get set on context elsewhere in codebase.  Is there some other pathway in which it could be set?